engine/classes/ElggUser.php

```diff
@@ -459,14 +459,6 @@ public function delete($recursive = true) {
 	 * @see \Elgg\Traits\Entity\PluginSettings::setPluginSetting()
 	 */
 	public function setPluginSetting(string $plugin_id, string $name, $value): bool {
```

Comment:
this function can be removed if the trait usage is changed to the default

Reply:
ok